adapter/config/default_config.go

```diff
@@ -328,4 +319,16 @@ var defaultConfig = &Config{
 			},
 		},
 	},
+	Tracing: tracing{
+		Enabled: false,
+		Type:    "zipkin",
```

Comment:
currently for router + enforcer to support publishing jaeger compatible traces. we have to configure jaeger with zipkin compatibility mode. therefore using zipkin exporter by default